frontend/src/components/Search/Search.jsx

```diff
@@ -80,63 +22,15 @@ export default function Search() {
                         type="text"
                         placeholder="Search for a Stock..."
                         value={searchInput}
-                        onChange={handleSearchChange}
+                        onChange={(event) => setSearchInput(event.target.value)}
                     />
                     <button type="submit">
                         <i className="material-icons">search</i>
                     </button>
                 </form>
-                {
-                    showDropdown && 
-                    <div className="dropdown">
-                        {
-                            searchResults.map((stock) => {
-                                if (isValidStock(stock)) {
-                                    return (
-                                        <Link key={stock.instrument_name} to={`/search/stocks/${stock.symbol}`} className="stock-link">
-                                            <div className="dropdown-item">
-                                                <p>{stock.instrument_name} ({stock.symbol})</p>
-                                            </div>
-                                        </Link>
-                                    )
-                                }
-                            })
-                        }
-                    </div>
-                }
+                <SearchQuery searchInput={searchInput} />
             </div>
-            <table className="search-table">
```

Comment:
nice, making these child components really improves this component's organization and readability

Reply:
+1